service.go

```diff
@@ -80,7 +80,7 @@ type Config struct {
 
 // ListenerConfig contains information needed to initialize the Listener Client service.
 type ListenerConfig struct {
-	Config chrysom.ListenerClientConfig
+	ListenerClientConfig chrysom.ListenerClientConfig
```

Comment:
```suggestion
	Client chrysom.ListenerClientConfig
```
This seems like the most relevant name, since `ListenerConfig` will already be included as the parent struct.